Add BYOC Docs (#1011)
okteto/docs: src/content/byoc/index.mdx

---

> title: Okteto Bring Your Own Cluster (BYOC)
> description: Introduction to Okteto Bring Your Own Cluster (BYOC)

Reviewer: ```suggestion
title: Okteto Bring Your Own Cloud (BYOC)
description: Introduction to Okteto Bring Your Own Cloud (BYOC)
```

---

> Okteto’s Bring Your Own Cluster (BYOC) offering allows you to run Okteto on your own cloud infrastructure while still enjoying the benefits of a fully managed experience.

Reviewer: ```suggestion
Okteto’s Bring Your Own Cloud (BYOC) offering allows you to run Okteto on your own cloud infrastructure while still enjoying the benefits of a fully managed experience.
```

---

> title: Okteto Bring Your Own Cloud (BYOC)

Reviewer 2: do you think we shoudl also mention here that BYOC can be purchased from the Marketplaces?

Author: Yes, that would be a good call out to have. Maybe I'll add it once the AWS listing is live so we can link to it?

description: Introduction to Okteto Bring Your Own Cloud (BYOC)
---

Okteto’s Bring Your Own Cloud (BYOC) offering allows you to run Okteto on your own cloud infrastructure while still enjoying the benefits of a fully managed experience.

> This is ideal for teams who need to meet strict security or compliance requirements, require custom infrastructure configurations, or simply want to maintain full control over their environments with a SaaS-like experience.

Reviewer 2: I think we should also mention high scale

We currently support BYOC on:

- [**Amazon Web Services (AWS)**](byoc/aws/index.mdx)

> - [**Google Cloud Platform (GCP)**](byoc/gcp/index.mdx)

Reviewer 2: do you think we should add a link for people that want BYOC on other platforms?

Author: Something like, "please contact us if you'd like to provision Okteto on other cloud providers?"

## How the BYOC Model Works

With BYOC, you bring the cloud provider account; we bring the platform and operational expertise.

- You provision and secure the cloud account

> - Okteto creates the Kubernetes cluster and installs all necessary components

Reviewer 2: We create the kubernetes cluster and othe infrastructure (vpcs, buckets, accounts, etc). Maybe we can instead say something like this?
```suggestion
- Okteto creates the all the necessary cloud infrastructure and software components
```
